travis: spurious feature_notifications.py failure

Happened on #19215

https://travis-ci.org/github/bitcoin/bitcoin/jobs/696258963#L2982

```
020-06-16T21:10:37.746000Z TestFramework (INFO): Initializing test directory /home/travis/build/bitcoin/bitcoin/ci/scratch/test_runner/test_runner_₿_🏃_20200616_205215/feature_notifications_69
29862020-06-16T21:10:42.422000Z TestFramework (INFO): test -blocknotify
29872020-06-16T21:10:42.964000Z TestFramework (INFO): test -walletnotify
29882020-06-16T21:10:43.544000Z TestFramework (INFO): test -walletnotify after rescan
29892020-06-16T21:10:46.212000Z TestFramework (INFO): test -walletnotify with conflicting transactions
29902020-06-16T21:11:49.282000Z TestFramework (ERROR): Assertion failed
2991Traceback (most recent call last):
2992  File "/home/travis/build/bitcoin/bitcoin/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/test_framework/test_framework.py", line 119, in main
2993    self.run_test()
2994  File "/home/travis/build/bitcoin/bitcoin/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/feature_notifications.py", line 101, in run_test
2995    self.sync_mempools()
2996  File "/home/travis/build/bitcoin/bitcoin/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/test_framework/test_framework.py", line 556, in sync_mempools
2997    sync_mempools(nodes or self.nodes, **kwargs)
2998  File "/home/travis/build/bitcoin/bitcoin/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/test_framework/util.py", line 485, in sync_mempools
2999    raise AssertionError("Mempool sync timed out after {}s:{}".format(
3000AssertionError: Mempool sync timed out after 60s:
3001  {'12c8a54d4b3c02291802f0b302bd0d5b39d0ce4e284684e9acf074937b11f1fd'}
3002  set()
```

--- a/test/functional/feature_notifications.py
+++ b/test/functional/feature_notifications.py
@@ -93,6 +93,7 @@ class NotificationsTest(BitcoinTestFramework):
             self.nodes[0].sethdseed(seed=self.nodes[1].dumpprivkey(keyhash_to_p2pkh(hex_str_to_bytes(self.nodes[1].getwalletinfo()['hdseedid'])[::-1])))
             self.nodes[0].rescanblockchain()
             self.nodes[0].generatetoaddress(100, ADDRESS_BCRT1_UNSPENDABLE)
+            self.sync_blocks()
 
             # Generate transaction on node 0, sync mempools, and check for
             # notification on node 1.

